docs/docs/examples/cookbooks/airtrain.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<p>Can you kindly do a string search and replace in this notebook to change "Llama Index" (with space) to "LlamaIndex" (without space).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/16332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done!</p>